test/acceptance/mixins/results.js

```diff
@@ -4,4 +4,10 @@ module.exports = (target) => {
   target.shouldBeOnResultsPage = function() {
     this.getText('h1').should.contain('records found for');
   };
+
+  target.clickFirstRecord = function() {
+    // browser.element gets just the first matching elements (browser.elements gets all matching ones)
+    const linkToFirstRecordDetails = browser.element('#records a').value.ELEMENT;
+    browser.elementIdClick(linkToFirstRecordDetails);
```

Comment:
s/browser/this/

Does this.click('#records a') definitely not work? Is the ".value.ELEMENT" stuff necessary?